data/redirects.json

```diff
@@ -17,10 +17,6 @@
     "from":"mailing-list",
     "to":"http://lists.lrug.org/listinfo.cgi/chat-lrug.org"
   },
-  {
-    "from":"meeting-calendar",
-    "to":"https://www.google.com/calendar/ical/407h2dbfrht4g2jsm2m2veibu8@group.calendar.google.com/public/basic.ics"
-  },
```

Comment:
let's leave in the redirect, but redirect to `meetings.ics` instead of the google calendar - cool urls don't change and all that.